hledger-lib/Hledger/Reports/AccountTransactionsReport.hs

```diff
@@ -159,28 +168,27 @@ pshowTransactions = pshow . map (\t -> unwords [show $ tdate t, T.unpack $ tdesc
 -- using the provided user-specified report query, a query specifying
```

Comment:
"transactions" -> "transactions (along with their original dates)" ?
Why must we provide these dates, are they different from what's in the transactions ?

Reply:
Yes, it is the minimum of the posting dates which match both `thisacctq` and `reportq`, or `tdate` if there are no matching postings.

Reply:
What do you think about
```
-- | Generate transactions report items from a list of transactions,
-- using the provided user-specified report query, a query specifying
-- which account to use as the focus, a starting balance, and a sign-setting
-- function.
-- Each transaction is accompanied by the date that should be shown for it 
-- in the report, which is not necessarily the transaction date; it is
-- the earliest of the posting dates which match both thisacctq and reportq, 
-- otherwise the transaction's date if there are no matching postings.
```